### DISCUSSION TITLE
Budibase update url's/ip's

### DISCUSSION THREAD

**Discussion by mojoaar** [Budibase/budibase#6086] [Help] [answered]
Hi,

We use a corporate proxy at work which means that budibase cannot check for upgrades and does a timeout error. Since there is no support for configuring a proxy in the app I need to try and whitelist the urls. Can anyone supply me with the cnames/ip's that budibase communicates with in regards to these update checks so I can get them whitelisted?

The only thing I can find in docs is a whitelisten ip but I think that is for inbound communication?

**Comment by Rory-Powell** [comment, marked as answer]
Hi @mojoaar, budibase should support proxies at the os level, we had some support added for this recently
https://github.com/Budibase/budibase/pull/5497

However in this specific case I believe the issue you are running into is that the `watchtower` container failing to reach dockerhub to check for or pull new images. There are some threads on their github issues relating to proxies
https://github.com/containrrr/watchtower/issues/394
https://github.com/containrrr/watchtower/issues/922

These don't seem overly useful though. Failing that, you might need to whitelist the dockerhub registry, there are some details here on how to do that
https://jfrog.com/knowledge-base/whitelist-the-following-if-your-docker-registry-is-behind-firewalls-and-proxy-servers/

**Comment by mojoaar** [comment]
@Rory-Powell Thank you very much for the information provided, much appriciated. We'll have a closer look monday, will provide info here with what we find.
